packages/common/components/content/blocks/item.marko

```diff
@@ -5,53 +5,61 @@ $ const primarySection = getAsObject(content, 'primarySection');
 $ const company = getAsObject(content, 'company');
 $ const imageLinkAttrs = { title: content.shortName, ...input.imageLinkAttrs };
 
-<endeavor-item modifiers=input.modifiers flush=input.flush>
-  <@image
-    display=input.withImage
-    height=input.imageHeight
-    image=content.primaryImage
-    lazyload=input.imageLazyload
-    link-attrs=imageLinkAttrs
-    link-to=content
-    min-height=input.imageMinHeight
-    min-width=input.imageMinWidth
-    modifiers=input.imageModifiers
-    options=input.imageOptions
-    position=input.imagePosition
-    use-placeholder=input.imageUsePlaceholder
-    width=input.imageWidth
-  />
-  <@header-left|{ block }|>
-    <cms-website-section-name tag="span" block=block obj=primarySection link=true />
-    <if(input.withType)>
-      <cms-text-element tag="span" block=block obj=content field="typeTitled" />
+<if(content.type === 'contact')>
```

Comment:
Was the contact if statement here before?

Reply:
It wasn't, but IMO it should be?

Reply:
👍